src/Essentials/samples/Samples/Platforms/Windows/Package.appxmanifest

```diff
@@ -60,11 +60,6 @@
 
   <Capabilities>
     <rescap:Capability Name="runFullTrust" />
-    <Capability Name="internetClient" />
-    <uap:Capability Name="contacts"/>
-    <DeviceCapability Name="location" />
-    <DeviceCapability Name="microphone"/>
-    <DeviceCapability Name="webcam"/>
```

Comment:
No permissions are needed in manifests for windows apps anymore - nor are there permission popups.